ansible/roles/de-services-common/tasks/main.yaml

```diff
@@ -1,27 +1,11 @@
 ---
 # tasks for services-common role
-
-- name: add the iplant group
-  sudo: yes
-  group: name=iplant state=present
-  tags:
-      - init_users
-      - init_iplant_user
-
-- name: add the iplant user
-  sudo: yes
-  user: name=iplant shell=/bin/bash groups=iplant append=yes
-  tags:
-      - init_users
-      - init_iplant_user
-
-- name: add the docker group to iplant user
+- name: add the current user to the docker group
   sudo: yes
-  user: name=iplant groups=docker append=yes
+  user: name={{ ansible_ssh_user }} groups=docker append=yes
```

Comment:
You are reading it right, and boy is it convenient!!
And no, it isn't directly needed by the deployment. I'll remove it.